### ISSUE TITLE
Error when creating Storage Account on Azure

### ISSUE THREAD

**Issue by simonwendel** [pulumi/pulumi-azure#488]
When following the quick-start (tried out both TypeScript and C# versions) I am unable to create a new storage account on Azure. All attempts result in something like the following error:

```
::azure:storage/account:Account::storage: Error reading static website for AzureRM Storage Account "storage76526422": accounts.Client#GetServiceProperties: Failure responding to request: StatusCode=403 -- Original Error: autorest/azure: Service returned an error. Status=403 Code="AuthorizationPermissionMismatch" Message="This request is not authorized to perform this operation using this permission.\nRequestId:<...>
```
I tried using the [quick-start example](https://www.pulumi.com/docs/get-started/azure/review-project/) verbatim and it still fails.

The actual resources are created in Azure, but the stack update is marked as failed at the [Pulumi dashboard](https://app.pulumi.com). Issuing ```pulumi destroy``` will successfully remove them again.

I use Pulumi v1.12.0

**Comment by mikhailshilkov** [comment]
Looks like https://github.com/terraform-providers/terraform-provider-azurerm/issues/6028 which should be fixed with the next Terraform release

**Comment by Odonno** [comment]
v2.1.0 is out just now ;)

**Comment by simonwendel** [comment]
Cool, knowing absolutely nothing about how Pulumi works, is it a big thing integrating the new release of the Terraform provider?

**Comment by jonathh21** [comment]
I got exactly this error too, not sure if it is bad timing.

How do i get this the update that fixes this?

**Comment by mikhailshilkov** [comment]
#489 is on the way

**Comment by jonathh21** [comment]
Thanks!

How does a fix like this manifest itself? A minor version increase in Pulumi actual, or because it is specific to Azure, is it a provider update that transparently comes down?

**Comment by mikhailshilkov** [comment]
The issue should be fixed in v2.1.0. Please report back if you still have issues.

**Comment by simonwendel** [comment]
Thanks for the super fast response, now it works perfectly with storage accounts.
